### DISCUSSION TITLE
Multiple Listener for cascading multiple select box

### DISCUSSION THREAD

**Discussion by joebakaz** [orchidsoftware/platform#2570] [Q&A] [answered]
Hi, I wish to know is there a way to do multiple listener within a screen. For example, i have 3 level category.
choosing main category will cascade to sub category, however sub category unable to cascaded to sub sub category.

any advice?

**Comment by joebakaz** [comment, marked as answer]
I have found my solution. Thanks.

**Comment by yetki** [comment]
I wish you shared your solution it would`ve helped me now
